scripts/test_generation.py

```diff
@@ -29,7 +29,7 @@ def generate_solve_plot(
 
     print(f"solving time: {time.time() - solution_start}")
 
-    plot_path(maze, path, show=True)
+    MazePlot._plot_path(maze, path, show=True)
```

Comment:
I'm not sure why this method is private now but it was throwing an error so I fixed

Reply:
I think the new interface by @canrager lets us add paths (with formatting) in a more structured way, and then just call `MazePlot().show()`

Note that for testing purposes, tests should call `MazePlot().plot()` and notebooks should call `plt.show()` on a separate line

Reply:
I don't have enough context to understand but I assume it's okay for me to go ahead and fix this

Reply:
The new interface sounds great. Can should remove any redundant methods and their tests

Reply:
the `.plot()` method does call the `_plot_maze()`, so I think the tests should stay. surprised this wasnt caught on his PR!